src/permission/compat.py

```diff
@@ -26,6 +26,19 @@ def isiterable(x):
 except ImportError:
     from django.utils.importlib import import_module
 
+if django.VERSION >= (1, 11):
```

Comment:
The `is_authenticated` and `is_anonymous` has become attributes from Django 1.10 (internally, it is a property which returns a callable boolean for backward compatibility [ref](https://github.com/django/django/blob/1.10/django/contrib/auth/base_user.py#L102))

So the line should be the following even if the method style is supported in Django 1.10

`if django.VERSION >= (1, 10):`

https://docs.djangoproject.com/en/1.10/ref/contrib/auth/#django.contrib.auth.models.User.is_authenticated